values.yaml

```diff
@@ -1,402 +1,134 @@
-# storageClassName -- Sets the storage class for all Coder services and user
-# environments. By default, the storageClassName is unspecified, and thus the
-# default StorageClass is used. If storageClassName is not specified and a
-# default StorageClass does not exist, then the deployment will fail. The
-# storageClass MUST support the ReadWriteOnce access mode.
-storageClassName: ""
-
-# serviceType -- See the following for the different serviceType options and
-# their use:
-# https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types
-serviceType: "ClusterIP"
-
-# podSecurityPolicyName -- The name of the pod security policy to apply to all
-# Coder services and user environments. The optional ingress has its own field
-# for pod security policy as well.
-podSecurityPolicyName: ""
```

Comment:
[PodSecurityPolicy has been deprecated](https://kubernetes.io/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/) and [should be replaced with custom admission controllers](https://www.antitree.com/2020/11/pod-security-policies-are-being-deprecated-in-kubernetes/), so I think it's fine to remove the annotation from our charts. We might want to contact the affected customers to inform them of the change, though.